### PR TITLE
Tutorial 2

### PR DESCRIPTION
**Pull request by Marty99-dev**

### CODE REVIEW

**Review by JorisMonnet — APPROVED** [review]
We need to put models/build in gitignore

File: app/src/main/res/layout/fragment_demonstration_tutorial.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
+    <Button
+        android:id="@+id/button"
```
Comment (CHOCOLVT) [inline app/src/main/res/layout/fragment_demonstration_tutorial.xml:9]: name not explicit

**Review by CHOCOLVT — APPROVED** [review]
need some improvement on interface but great to have music play

**Comment by JorisMonnet** [comment]
Need some tests ^^'

**Pull request closed without merging**